src/control/cmd/daos_server/ms_recovery_test.go

```diff
@@ -0,0 +1,43 @@
+//
+// (C) Copyright 2024 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package main
+
+import (
+	"testing"
+
+	"github.com/daos-stack/daos/src/control/logging"
+)
+
+// TestDaosServer_MS_Commands_JSON verifies that the JSON-output flag is disabled for
+// management-service command syntax.
+func TestDaosServer_MS_Commands_JSON(t *testing.T) {
+	log := logging.NewCommandLineLogger()
+
+	runJSONCmdTests(t, log, []jsonCmdTest{
+		{
+			"MS status; JSON",
+			"ms status -j",
```

Comment:
It's fine to not do it as part of this PR, but I think this command should support JSON output for consumption by other tools.

Reply:
https://daosio.atlassian.net/browse/DAOS-15536 If you could give some guidance on the ticket I found it difficult to see how all the different fragments of information that are written to standard out would be framed in a single JSON response. TIA